### ISSUE TITLE
Traffic Shaper: Introduce delay

### ISSUE THREAD

**Issue by mimugmail** [opnsense/core#2291]
Hi,

while searching for better help texts for masks as discussed with @AdSchellevis in IRC I stumbled upon `delay` in pipes. I'm curious why nobody asked for this yet since it's a really easy way to slow down unwanted traffic .. it could also be an instant throttle since it's callable via API.

Since it's just a single field for template I'm asking you guys if I can send you a PR for this and what you think in general about this?

http://info.iet.unipi.it/~luigi/dummynet/

```
    ipfw add pipe 3 out
    ipfw add pipe 4 in
    ipfw pipe 3 config bw 128Kbit/s queue 10 delay 1000ms
    ipfw pipe 4 config bw 640Kbit/s queue 30 delay 1000ms 
```

**Comment by AdSchellevis** [comment]
Hi Michael,

Sure you can, there's no reason to not include the delay option, although you may need to test it first to know for sure it's functional in FBSD. I remember the burst option a long time ago (https://github.com/opnsense/core/issues/226).

Best regards,

Ad

**Comment by mimugmail** [comment]
Closed via #2297
